src/main/java/com/github/ferstl/maven/pomenforcers/CompoundPedanticEnforcer.java

```diff
@@ -48,7 +48,8 @@
  *         &lt;dependencyManagementArtifactIdPriorities&gt;commons-,utils-&lt;/dependencyManagementArtifactIdPriorities&gt;
  *         &lt;!-- DEPENDENCY_CONFIGURATION configuration --&gt;
  *         &lt;manageDependencyVersions&gt;true&lt;/manageDependencyVersions&gt;
- *         &lt;allowUnmangedProjectVersions&gt;true&lt;/allowUnmangedProjectVersions&gt;
+ *         &lt;allowUnmanagedProjectVersionsInDependencies&gt;true&lt;/allowUnmanagedProjectVersionsInDependencies&gt;
+ *         &lt;allowedUnmanagedProjectVersionPropertiesForDependencies&gt;version,project.version&lt;/allowedUnmanagedProjectVersionPropertiesForDependencies&gt;
```

Comment:
Your new (non-deprecated) `allowUnmanagedProjectVersions` and  `allowedUnmanagedProjectVersionProperties` are not documented here and will not be shown in the wiki.

Unless of course you don't want to encourage its use. But then why add it in the first place ;)

Reply:
Yes, I‘m aware of that. But I had to decide what to show in the example. Having both properties does not make sense. The allowUnmanagedProjectVersions and allowedUnmanagedProjectVersionProperties properties will show up in the Wiki as well, but not so prominently.
The example shows the capabilities of the enforcer, but the two generic properties may be more convenient for use.

Reply:
That's fine. Only pointed it out in case you missed it ;).